### PR TITLE
feat: workspace name is displayed in toolbar

### PR DESCRIPTION
**Pull request by mykytabatrak**

### CODE REVIEW

**Review by ursarik — APPROVED** [review]

File: backend/src/api/routes/workspace-route.ts [hunk 1]
```diff
@@ -40,6 +41,7 @@ router
     verifyWorkspaceId,
     permit(RoleType.ADMIN),
     run((req: IRequestWithUser) => getWorkspaceUsers(req.workspaceId)),
-  );
+  )
+  .get('/workspace/:id', run((req) => getOne(req.params.id, req.userId)));
```
Comment (aleksandr-kuharenko) [inline backend/src/api/routes/workspace-route.ts:45]: You already have "/api/workspaces" [here](https://github.com/BinaryStudioAcademy/bsa-2021-infostack/blob/e01259c5aee6a9dbbc79e9bd7cf852360bae4b93/backend/src/api/routes/index.ts#L11)

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]
Fix endpoint

File: backend/src/api/routes/workspace-route.ts (hunk 1 above)
Comment (mykytabatrak) [reply backend/src/api/routes/workspace-route.ts]: "api/workspaces/:id" is already in use, should I repurpose it?

Comment (aleksandr-kuharenko) [reply backend/src/api/routes/workspace-route.ts]: As I understood, `api/workspaces/:id` currently is used to get all workspaces for the user. And the Id in endpoint is userId. It should not be like this...
Move this functionality to `api/users/:id/workspaces` endpoint. 
And use `api/workspaces/:id` to get workspace by id

File: backend/src/api/routes/workspace-route.ts [hunk 2]
```diff
@@ -15,31 +16,19 @@ import { verifyWorkspaceId } from '../middlewares/verify-workspace-id';
...
 router
-  .get(
-    '/',
-    run((req) => getAll(req.userId)),
-  )
-  .post(
-    '/',
-    run((req) => create(req.userId, req.body)),
-  )
+  .get('/', run(req => getUserWorkspaces(req.userId)))
```
Comment (ursarik) [inline backend/src/api/routes/workspace-route.ts:19]: why do we need this if we have a route `/api/users/:id/workspaces` which does the same

**Review by ursarik — APPROVED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**